common/unsafe/src/main/java/org/apache/spark/unsafe/array/ByteArrayMethods.java

```diff
@@ -91,4 +93,39 @@ public static boolean arrayEquals(
     }
     return true;
   }
+
+  public static boolean contains(byte[] arr, byte[] sub) {
```

Comment:
I assume these code are adjusted from UTF8String

Reply:
Yea, the method from UTF8String is optimized.